ctapipe/io/hdf5eventsource.py

```diff
@@ -70,6 +70,7 @@
 COMPATIBLE_DATA_MODEL_VERSIONS = [
     "v4.0.0",
     "v5.0.0",
+    "v6.0.0",
```

Comment:
Is this true? Can we read the old files still?

I guess we should raise a blank error when detecting an older version, but only raise a warning when someone tries to read the old disp format?

We shouldn't remove the option to read DL1 files for this relatively minor change in DL2 that probably noone is using together with the HDF5EventSource.

Reply:
Until now, the DispContainer was not loaded by HDF5EventSource. I will add it there now and there should be now issues regarding compatibility.